packages/firebase_messaging/ios/Classes/FirebaseMessagingPlugin.m

```diff
@@ -180,4 +150,10 @@ - (void)application:(UIApplication *)application
   [_channel invokeMethod:@"onIosSettingsRegistered" arguments:settingsDictionary];
 }
 
+// this method replaces the deprecated 'tokenRefreshNotification' method
```

Comment:
I'd remove this comment. That's not really valuable to anybody looking at the code in the future.

Reply:
Yup, good call.